File: Drivers/FSCC Drivers/windows/lib/fscc/c++/examples/memory-cap.cpp
#include <fscc.hpp> // Fscc::Port

int main(void)
{
    Fscc::MemoryCap memcap;
    Fscc::Port p(0);

    memcap.input = 1000000; // 1 MB
    memcap.output = `000000; // 1 MB

    p.SetMemoryCap(memcap);
    memcap = p.GetMemoryCap();

    return 0;
}

File: Drivers/FSCC Drivers/windows/lib/fscc/net/examples/memory-cap.cs
using Fscc;

public class Tutorial
{
    public static int Main(string[] args)
    {
        Fscc.Port p = new Fscc.Port(0);
    
        p.MemoryCap.Input = 1000000; // 1 MB
        p.MemoryCap.Output = 1000000; // 1 MB
        
        var input = p.MemoryCap.Input;
        var output = p.MemoryCap.Output;

        return 0;
    }
}